plugins/org.komodo.shell/src/org/komodo/shell/commands/core/CreateCommand.java

```diff
@@ -385,12 +385,20 @@ public int tabCompletion( String lastArgument,
 
             return -1;
         } else if ( getArguments().size() == 2 ) {
-            // TODO need to handle those types that have args after name and before parent path
-            // The arg is expected to be a path
-            updateTabCompleteCandidatesForPath( candidates, getWorkspaceStatus().getCurrentContext(), true, lastArgument );
-
-            // Do not put space after it - may want to append more to the path
-            return NO_APPEND_SEPARATOR;
+        	final String typeArg = getArguments().get(0);
+        	// VDB Translator - candidate is translator type
+        	if( (lastArgument == null) && typeArg.equals(KomodoType.VDB_TRANSLATOR.getType())) {
```

Comment:
I was thinking we could offer foreign key tab-completion for the table reference. Wdyt?